packages/components/src/Radio/Radio/_docs/Radio.stickersheet.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Radio } from "../index"
+
+export default {
+  title: "KAIO-staging/Radio controls/Radio",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `RadioField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Header
+        headings={["Default", "Disabled", "Focus", "Hover"]}
+      />
+      <StickerSheet.Body>
+        <StickerSheet.Row rowTitle="Off">
+          <Radio
+            id="radio1"
+            name="radio1"
+            value="radio1"
+            reversed={isReversed}
+          />
+          <Radio
+            id="radio2"
+            name="radio2"
+            value="radio2"
+            reversed={isReversed}
+            disabled
+          />
```

Comment:
Does this actually house the disabled styles? The stickersheet doesn't seem to show it looking different